lib/cldr.js

```diff
@@ -2091,6 +2091,43 @@ Cldr.prototype = {
 
     return weekData;
   },
+
+  extractTTSCharacterLabels(localeId) {
```

Comment:
Subjective nit: Could we expand `TTS` to `TextToSpeech` in these function names?